Source/VisualStudioTools/Private/VisualStudioToolsCommandlet.cpp

```diff
@@ -380,6 +381,24 @@ static void RunAssetScan(
 			FObjectPropertyBase::GetExportPath(Class.Get(), nullptr /*Parent*/, nullptr /*ExportRootScope*/, 0 /*PortFlags*/));
 		});
 
+	// Take account of any core redirects for the blueprint classes we want to scan.
+	for (const auto& BaseClass : FilterBaseClasses)
+	{
+		if (BaseClass.IsValid())
+		{
+			TArray<FCoreRedirectObjectName> PreviousNames;
+			if (FCoreRedirects::FindPreviousNames(ECoreRedirectFlags::Type_Class, BaseClass->GetPathName(), PreviousNames))
+			{
+				for (const auto& PreviousName : PreviousNames)
+				{
+					// FString PreviousString = FObjectPropertyBase::GetExportPath(BaseClass->GetClass()->GetClassPathName(), PreviousName.ToString()); // Alternative way to add /Script/CoreUObject.Class'' wrapper - but not sure it makes sense to use the new class when referencing a previous name
+					FString PreviousString = "/Script/CoreUObject.Class'" + PreviousName.ToString() + "'";
```

Comment:
I tried to find a helper to format the class name as an instance of `UClass` but did not find any, so this should be fine for most cases (maybe not if someone is using classes at runtime that are subclasses of `UClass`)